arelle/ModelDocument.py

```diff
@@ -419,14 +419,20 @@ def create(modelXbrl, type, uri, schemaRefs=None, isEntry=False, initialXml=None
         Xml = '<nsmap>{}{}</nsmap>'.format(initialComment or '', initialXml or '')
     elif type == Type.INSTANCE:
         # modelXbrl.uriDir = os.path.dirname(normalizedUri)
+        instance_namespace = '<xbrl xmlns="http://www.xbrl.org/2003/instance"'
+        if defaultNamespace:
+            instance_namespace = f'<{defaultNamespace}:xbrl xmlns:{defaultNamespace}="http://www.xbrl.org/2003/instance"'
```

Comment:
nit: use an else block (or ternary) both here and below for the closing tag:
```suggestion
        if xbrliNamespacePrefix:
            instance_namespace = f'<{xbrliNamespacePrefix}:xbrl xmlns:{xbrliNamespacePrefix}="http://www.xbrl.org/2003/instance"'
        else:
            instance_namespace = '<xbrl xmlns="http://www.xbrl.org/2003/instance"'
```